Session-4/CurrencyExchangeApp/CurrencyConverter/CurrencyConverter.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Configuration;
+using RestSharp;
+using System.Collections.Generic;
+using System.IO;
+
+
+namespace CurrencyConverter
+{
+    public class CurrencyExchange
+    {
+        public void GetApidata()
```

Comment:
Fix the issues method in session 3 here  as well.
This method is not meaningful. Don't do multiple things in a single method. Separate file writing logic with getting data from api.